### ISSUE TITLE
about the pretrain model "90k" and "90k_bn"

### ISSUE THREAD

**Issue by irfanICMLL** [hellochick/ICNet-tensorflow#14]
I have evalute the miou of these two pre train model.
The results are as follows:
90k: 78.56%
90k_bn: 70.34%
Where does the pretrain model "90k_bn" come from?
Can I fune-tuning the "90k" model?
Thanks a lot for your  implementation, and looking forword for your reply~~:)

**Comment by irfanICMLL** [comment]
Now I know, the "90k_bn" is the model trained on your own dataset, right?

**Comment by hellochick** [comment]
Hey @irfanICMLL .
The pretrained model "90k_bn" comes from the source code of ICNet:
https://github.com/hszhao/ICNet/tree/master/evaluation/model
You can fine-tune on it, but I think it cannot be much better.

The reason is, this model have been already pruned by the original author. So, if you focus on the number of kernels, you will find all the numbers of kernels are half of those in PSPNet.

So the best approach is, fine-tuning on the model before pruning, and then do model compression.
Hope my answer can help you. Btw, I  keep training ADE20k dataset on it, and get result in the bottom of README.

**Comment by hellochick** [comment]
@irfanICMLL , I don't think the decrease of the accuracy is caused by the pruning, I still can't find the answer to the strange accuracy. Maybe the model is different to each other.
You can directly modified `train.py`, I have create the loss formula in it. You can take a look on it.

**Comment by chuchienshu** [comment]
Hi, @hellochick I do not have a good command of transfer the caffemodel's weight to tensorflow's weight.would you mind share your transfer script?And to the best information I known, only the weights that  have the exactly same name  could be assignment in tensorflow. How do you solve this problem?Thanks a lot!
